components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -270,7 +270,8 @@ def _unwrapunits(self, obj, units=None):
             return obj
         return obj.getValue()
 
-    def _getQueryString(self, opts=None):
+    @classmethod
+    def _getQueryString(klass, opts=None):
```

Comment:
This is probably a historical (i.e. pre-PEP8) Josh/Carlos-style thing which should be dropped in favor of `cls`.